book/src/install.md

```diff
@@ -1,179 +1,240 @@
-# Installation
+# Installing Helix
 
-We provide pre-built binaries on the [GitHub Releases page](https://github.com/helix-editor/helix/releases).
+<!--toc:start-->
+- [Installing Helix](#installing-helix)
+  - [Using the Pre-built Binaries](#using-the-pre-built-binaries)
+  - [Installing Helix on Linux through the Official Package Manager](#installing-helix-on-linux-through-the-official-package-manager)
+  - [Installing Helix on Linux via Third-Party Repositories](#installing-helix-on-linux-via-third-party-repositories)
+  - [Installing Helix on macOS via Homebrew](#installing-helix-on-macos-via-homebrew)
+  - [Installing Helix on Windows](#installing-helix-on-windows)
+  - [Building from Source](#building-from-source)
+  - [Validating the Installation](#validating-the-installation)
+  <!--toc:end-->
 
-[![Packaging status](https://repology.org/badge/vertical-allrepos/helix.svg)](https://repology.org/project/helix/versions)
+To install Helix, follow the instructions specific to your operating system.
+Additionally:
 
-## OSX
+- To get the latest pre-release version of Helix, you will need to
+  [build from source](#building-from-source).
 
-Helix is available in homebrew-core:
+- To take full advantage of Helix, install the language servers for your
+  preferred programming languages. Refer to the
+  [Helix Wiki](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers)
+  for detailed instructions.
 
-```
-brew install helix
-```
+## Using the Pre-built Binaries
 
-## Linux
+Download pre-built binaries from the
+[GitHub Releases page](https://github.com/helix-editor/helix/releases). You will
+need to add the binary to your system's `$PATH` to access it from the command
+line.
 
-### NixOS
+## Installing Helix on Linux through the Official Package Manager
 
-A [flake](https://nixos.wiki/wiki/Flakes) containing the package is available in
-the project root. The flake can also be used to spin up a reproducible development
-shell for working on Helix with `nix develop`.
+If your Linux distribution has Helix available through its official package
+manager, install it through that. The following shows availability
+throughout the Linux ecosystem:
 
-Flake outputs are cached for each push to master using
-[Cachix](https://www.cachix.org/). The flake is configured to
-automatically make use of this cache assuming the user accepts
-the new settings on first use.
+[![Packaging status](https://repology.org/badge/vertical-allrepos/helix.svg)](https://repology.org/project/helix/versions)
```

Comment:
I didn't spot that Windows and macOS users could use the list of repositories. This makes the Windows and macOS specific sections redundant. I have modified the heading to be `Official repositories for Linux, macOS and Windows` and added some more info. This makes the whole document much more concise and logical, excellent! It then makes more sense to differentiate between 'official' and 'third party Linux' repositories as there are now only the two headings, with 3rd party having subheadings.

What do you think of it now?

Reply:
I would keep the homebrew/windows sections since it can help steer users not familiar with certain package managers (for example if you've never heard of Scoop you just assume it's not available on windows).

See wezterm for example again:
https://wezfurlong.org/wezterm/install/macos.html
https://wezfurlong.org/wezterm/install/windows.html

Reply:
I have added back the homebrew/windows sections and improved the document further by removing a few things and adding new subheadings.